src/quo/foundations/shadows.cljs

```diff
@@ -5,60 +5,59 @@
     [quo.theme :as quo.theme]
     [react-native.platform :as platform]))
 
+(defn clamp
+  [value min-val max-val]
+  (max min-val (min max-val value)))
+
+(defn get-shadow-color
+  ([color opacity]
+   (get-shadow-color color opacity 5))
+  ([color opacity coefficient]
+   {:shadow-color   (if platform/android?
+                      (colors/alpha color (clamp (* opacity coefficient) 0 1))
+                      color)
+    :shadow-opacity opacity}))
+
+(defn get-shadow-radius
+  ([radius]
+   (get-shadow-radius radius 0.8))
```

Comment:
Even if you arrived at the default coefficients by trial and error, their values won't be obvious for some dev in the future. Seems worth adding a docstring to `get-shadow-radius` and `get-shadow-color` mentioning something about these default coefficients.